database/migrations/2019_11_06_171938_create_ratings_and_users_table.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateRatingsAndUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
+            $table->unsignedInteger('id');
+            $table->string('slack_id', 10)->unique()->nullable();
+            $table->string('name_first');
+            $table->string('name_last');
+            $table->string('nickname')->nullable();
+            $table->string('email');
+            $table->string('password')->nullable();
+            $table->timestamp('password_set_at')->nullable();
+
+            $table->timestamp('last_login')->nullable();
+            $table->ipAddress('last_login_ip')->default(0);
+
+            $table->string('remember_token', 100)->nullable();
+
+            $table->timestamp('joined_at')->nullable();
+            $table->timestamp('cert_checked_at')->nullable();
+            $table->timestamps();
+            $table->softDeletes();
+        });
+
+        Schema::create('ratings', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
Probably don't need a `bigInteger` here, probably even an unsigned tiny integer would be enough?